crates/e2e/src/setup/colocation.rs

```diff
@@ -172,6 +171,7 @@ http-timeout = "10s"
 
 [submission]
 gas-price-cap = "1000000000000"
+max-confirm-time= "2s"
```

Comment:
Is this actually needed?
I would assume in a forked test we'd bring the tx on-chain immediately and don't even need this upper limit.